Production/govuk_ios/ViewModels/SettingsViewModel.swift

```diff
@@ -100,19 +103,36 @@ class SettingsViewModel: SettingsViewModelInterface {
         let rowTitle = String.settings.localized(
             "helpAndFeedbackSettingsTitle"
         )
+        let feedbackUrl = helpAndFeedbackUrl?.absoluteString ?? Constants.API.helpAndFeedbackUrl
+
         return LinkRow(
             id: "settings.helpAndfeedback.row",
             title: rowTitle,
             body: nil,
             action: { [weak self] in
                 self?.openURLIfPossible(
-                    urlString: Constants.API.helpAndFeedbackUrl,
+                    urlString: feedbackUrl,
                     eventTitle: rowTitle
                 )
             }
         )
     }
 
+    private var helpAndFeedbackUrl: URL? {
+        let model = deviceInformation.model
+        let systemVersion = deviceInformation.systemVersion
+        let device = "Apple \(model) \(systemVersion)"
+        let appVersion = versionProvider.fullBuildNumber ?? "-"
+
+        var feedbackUrl = URLComponents(string: Constants.API.helpAndFeedbackUrl)
+        feedbackUrl?.queryItems = [
+            .init(name: "app_version", value: appVersion),
+            .init(name: "phone", value: device)
+        ]
+
+        return feedbackUrl?.url
```

Comment:
Might get rid of line 106 and instead have this method return a string:

`return feedbackUrl?.url?.absoluteString ?? Constants.API.helpAndFeedbackUrl`

Reply:
Discussed away from PR, have implemented a tweaked suggestion: https://github.com/alphagov/govuk-mobile-ios-app/pull/211/commits/a64302852dd3462a1549dfd7ef4a6f4a4f35c7cf.